pkg/gds/emails/client.go

```diff
@@ -218,55 +205,49 @@ func (m *EmailManager) SendRejectRegistration(vasp *pb.VASP, reason string) (sen
 		Reason: reason,
 	}
 
-	var contacts = []*pb.Contact{
-		vasp.Contacts.Technical, vasp.Contacts.Administrative,
-		vasp.Contacts.Billing, vasp.Contacts.Legal,
-	}
-
 	// Attempt at least one delivery, don't give up just because one email failed
 	// Track how many emails and errors occurred during delivery.
 	var nErrors uint8
-	for idx, contact := range contacts {
-		// Skip any contacts that we can't send emails to
-		if contact == nil || contact.Email == "" {
+	iter := models.NewContactIterator(vasp.Contacts, false, true)
+	for iter.Next() {
+		var contact *pb.Contact
+		var kind string
+		if contact, kind, err = iter.Value(); err != nil {
+			log.Error().Err(err).Str("vasp", vasp.Id).Msg("could not retrieve contact info")
+			nErrors++
```

Comment:
With the suggestion for multi-error below; we don't have to track nErrors, we can just log the error at the end of the iteration if it occurred.